desktop/storeService.ts

```diff
@@ -1,11 +1,55 @@
-const Store = require('electron-store');
+import Store from 'electron-store';
+import { TypedSchemaStore } from '../shared/types';
 
 class StoreService {
-  static store: typeof Store;
+  public static store: Store<TypedSchemaStore>;
 
   static init() {
     if (!StoreService.store) {
-      StoreService.store = new Store();
+      StoreService.store = new Store<TypedSchemaStore>({
+        accessPropertiesByDotNotation: true,
+        schema: {
```

Comment:
Can we get rid of this `schema`? It is hardly maintainable.
Also, please, get rid of excessive properties.